.github/workflows/scheduled-idp-web-tests.yml

```diff
@@ -10,6 +10,35 @@ on:
     # https://docs.github.com/en/actions/reference/events-that-trigger-workflows#schedule
     # Do not change this without also changing stop-test-service-providers.yml
     - cron: '0 10 * * *'  # 3am PDT, 2am PST
```

Comment:
Thanks @goulter, I just have one small thing. I notice  that this is also referenced in `.github/workflows/old-scheduled-idp-web-tests.yml.` With both actions using similar cron expressions, could this result in them being triggered twice? 

If that's the case, we might not need to keep the old one, as it would still be easily retrievable via Git history if needed.

```yml
 - cron: '0 10 * * *'  # 3am PDT, 2am PST
```

Looking forward to hearing your thoughts! 😊